lib/taskana-core/src/test/java/acceptance/priorityservice/PriorityServiceAccTest.java

```diff
@@ -31,23 +30,21 @@
 @ExtendWith(JaasExtension.class)
 class PriorityServiceAccTest extends AbstractAccTest {
 
-  private static final TaskService TASK_SERVICE = taskanaEngine.getTaskService();
```

Comment:
I'm not sure here: isn't this the new way to provide the TASK_SERVICE? I'm afraid we might go back in time here...